content/en/developers/metrics/custom_metrics.md

```diff
@@ -5,224 +5,221 @@ aliases:
 
-## How is a custom metric defined ?
+There are no enforced [fixed rate limits][5] on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][6].
 
-A custom metric refers to a single, unique combination of a metric name, host, and any tags.
+## Allocation
 
-Custom metrics generally refer to any metric that you send using StatsD, [DogStatsD][1], or through extensions made to the [Datadog Agent][2]. Some [integrations][3] can potentially emit an unlimited number of metrics that can also count as custom. See [standard integrations emit custom metrics][4] for details.
+You are allocated a certain number of custom metrics based on your Datadog pricing plan:
 
-Use tags to fully leverage the capabilities of Datadog through scoping and alerting. When using tags, one submitted metric actually leads to **multiple unique tag combinations**- counting towards your custom metrics count.
+* **Pro**: 100 custom metrics per host
+* **Enterprise**: 200 custom metrics per host
 
-**Example 1**
+These allocations are counted across your entire infrastructure. For example, if you are on the Pro plan and licensed for three hosts, 300 custom metrics are allocated. The 300 custom metrics can be divided equally across each host, or all 300 metrics could be used by a single host.
 
-For example, suppose you are interested in measuring the average `temperature` in the US. You collect the following temperature measurements every 10 seconds for the past minute from Orlando, Miami, New York, Boston and Seattle. Each `temperature` measurement is tagged with the information about `city`, `state`, `region`, and `country`. 
+Using this example, the graphic below shows scenarios that do not exceed the allocated custom metric count:
 
-|  |  |  |  |  |  |  |  |
-|------------------------------|----|----|----|----|----|----|----|
-| Orlando, FL, Southeast, USA | 80 | 80 | 80 | 80 | 81 | 81 | 81 |
-| Miami, FL, Southeast, USA | 82 | 82 | 82 | 82 | 82 | 82 | 82 |
-| Boston, MA, Northeast, USA | 78 | 78 | 78 | 78 | 78 | 79 | 79 |
-| New York, NY, Northeast, USA | 79 | 79 | 79 | 79 | 79 | 79 | 79 |
-| Seattle, WA, Northwest, USA | 75 | 75 | 75 | 75 | 75 | 75 | 75 |
- 
- Each unique tag combination of `city`, `state`, `region`, and `country` represents a timeseries / custom metric. Using the five timeseries above, you can determine the average temperature in the US, Northeast, or Florida.
+{{< img src="developers/metrics/custom_metrics/Custom_Metrics_300.jpg" alt="Custom_Metrics_300" responsive="true" style="width:80%;">}}
 
-**Example 2**
+There are no enforced [fixed rate limits][5] on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][6].
```

Comment:
this is repetitive given it's mentioned in the overview too - was that intentional?

Reply:
I made some updates I forgot to push. Should be updated now.